gtsfm/common/two_view_estimation_report.py

```diff
@@ -34,9 +52,12 @@ class TwoViewEstimationReport:
         U_error_deg: relative translation error w.r.t. GT. Only defined if GT poses provided.
         i2Ri1: relative rotation.
         i2Ui1: relative translation direction.
+        configuration_type: classification of two-view relationship.
```

Comment:
maybe specify that configuration_type must be of Enum type `TwoViewConfigurationType`

Reply:
Thanks for the review Adi. I'm a bit hesitant to add that info inline since I think it might be redundant with the type hint in the function signature